e2e/cloud_manager/decryption_aws_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || (decrypt && (cloudmanager || om44 || om50))
+
+package cloud_manager_test
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongocli/e2e"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed decryption/aws/*
+var filesAWS embed.FS
+
+const awsTestsInputDir = "decryption/aws"
+
+func TestDecryptWithAWS(t *testing.T) {
+	req := require.New(t)
+
+	cliPath, err := e2e.Bin()
+	req.NoError(err)
+
+	tmpDir := t.TempDir()
+
+	t.Cleanup(func() {
+		err = os.RemoveAll(tmpDir)
+		req.NoError(err)
+	})
+
+	i := 1
+	t.Run(fmt.Sprintf("Test case %v", i), func(t *testing.T) {
```

Comment:
[q] do we expect more test cases?

Reply:
@gssbzn no we don't support multiple cases I've pushed a change

Reply:
I spotted that too - it's in line with decryption_kmip_test.go and decryption_gcp_test.go, so probably fine if more test cases will be added in future. Otherwise the i variable is redundant.